popUp-modal.css

```diff
@@ -0,0 +1,211 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap");
+
+* {
+  font-family: "Inter", sans-serif;
+}
+
+#popUp-modal {
+  display: none;
+  position: fixed;
+  z-index: 1000;
+  padding: 10px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: #c1c7d0b0;
+  box-sizing: border-box;
+}
+
+.content-modal {
+  margin: 5px auto;
+  width: 80%;
+  background-color: #fff;
+  border-radius: 10px;
+  padding: 20px;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
```

Comment:
The horizontal scroll is still available which does not give a good user experience and also makes your project not match the actual design. Kindly ensure that both scrolls are removed and your design matches the actual figma design 

![Screenshot from 2023-02-02 12-36-34](https://user-images.githubusercontent.com/65233860/216327063-f1275490-de98-440f-9101-b9f67969e0ce.png)

Reply:
##  Thank you for the feedback 

<img width="1280" alt="Screenshot 2023-02-02 at 15 55 21" src="https://user-images.githubusercontent.com/57327352/216330825-44f45a26-b6cd-4231-bc7c-b945a92f4dd8.png">

But when am seeing my screen there is no horizontal scroll and the two buttons are well positioned